src/smw/ai.cpp

```diff
@@ -179,7 +179,7 @@ void CPlayerAI::Think(COutputControl * playerKeys)
     } else if (nearestObjects.teammate) {
         actionType = 3;
     } else if (nearestObjects.goal) {
-        if (nearestObjects.playerdistance < 4096)
+        if (nearestObjects.playerdistance < 8100)
```

Comment:
Is there a particular reason this is 8100 (instead of eg. 8192 or any other number)?

Reply:
8100 is used [here](https://github.com/mmatyas/supermariowar/blob/ad06e2b4c3192534227d28aff5c59251c00c1395/src/smw/ai.cpp#L270) when running away from other players. Because of the way the code is structured, the AI would run away at either 8100 or 4096 depending on if there is a goal, like a powerup.